docs/faq.mdx

```diff
@@ -96,4 +96,14 @@ beforeEach(() => {
 })
 ```
 
+### Apollo Client 
+
+```js
+import { client } from '../lib/apolloClient';
+
+beforeEach(() => {
+  return client.cache.reset()
```

Comment:
Does `.reset()` return a Promise? Wonder if the return statement is necessary here.

Reply:
Yes, it does. Without returning the promise we would occasionally get a “Cannot reset cache with in-flight queries” error from Apollo.